pdns/backends/gsql/gsqlbackend.cc

```diff
@@ -1013,7 +1041,11 @@ bool GSQLBackend::get(DNSResourceRecord &r)
     } catch (SSqlException &e) {
       throw PDNSException("GSQLBackend get: "+e.txtReason());
     }
-    extractRecord(row, r);
+    try {
+      extractRecord(row, r);
+    } catch (...) {
+      goto skiprow;
```

Comment:
silent failure? shouldn't we SERVFAIL if we cannot understand what we are seeing?